models/zeroex/ethereum/zeroex_ethereum_api_fills.sql

```diff
@@ -457,60 +438,60 @@ SELECT
         ms.symbol AS maker_symbol,
         CASE WHEN lower(ts.symbol) > lower(ms.symbol) THEN concat(ms.symbol, '-', ts.symbol) ELSE concat(ts.symbol, '-', ms.symbol) END AS token_pair,
         taker_token_amount_raw / pow(10, tp.decimals) AS taker_token_amount,
-        taker_token_amount_raw,
+        cast(taker_token_amount_raw as int256) as taker_token_amount_raw,
         maker_token_amount_raw / pow(10, mp.decimals) AS maker_token_amount,
-        maker_token_amount_raw,
+        cast(maker_token_amount_raw as int256) as maker_token_amount_raw,
```

Comment:
it looks like you reverted back to `int256` for raw amounts all throughout. is there any reason why? can we at minimum cast at the very end here to `uint256`?

i'm afraid when we merge into `dex.trades` it'll fail in the union on different data types